tests/test-project/src/test/webview/webviewView.test.ts

```diff
@@ -52,7 +54,7 @@ describe('WebviewViews', function () {
 				this.timeout(10000);
 				await new Workbench().executeCommand(param.command);
 				await new Promise((res) => setTimeout(res, 2000));
-				webviewView = new WebviewView();
+				webviewView = new WebviewView(param.container());
```

Comment:
the selection criteria will select Welcome area first if this test is run separately. not sure if this is better to make the selection criteria more exact?